geode-core/src/main/java/org/apache/geode/cache/execute/FunctionContext.java

```diff
@@ -98,4 +99,6 @@ default String getMemberName() {
 
     return member.getId();
   }
+
+  DistributedSystem getDistributedSystem();
```

Comment:
This needs javadoc since it's a public API addition.  Under what conditions will it return null?  If possible I think it would be best not surprise a caller with a null return.

Reply:
Hi @metatype , I have updated PR with javadocs.
Sorry for confusion,  this API will not return null, but one can get the DistributedSystem using cache also, i.e., FunctionContext.getCache().getDistributedSystem(). But I found that cache in FunctionContext can be null, so added a new API.